base/int.jl

```diff
@@ -44,6 +44,22 @@ const BitUnsigned64T   = Union{Type{UInt8}, Type{UInt16}, Type{UInt32}, Type{UIn
 
 const BitIntegerType = Union{map(T->Type{T}, BitInteger_types)...}
 
+# interconvert equilength BitInteger types
+
+unsigned(::Type{Int8}) = UInt8
```

Comment:
These should already be defined. Why is this necessary?

Reply:
`unsigned(::Type{IntN}) = UIntN` are defined in `Base/multinverses.jl`.
`signed(::Type{UIntN}) = IntN` are not defined.
I thought the two sets of definitions belonged in the same place, and that `Int.jl` was more appropriate than `multinverses.jl`.  I moved the defs from `multinverses.jl` to `Int.jl` and added the defs for `signed(_)` there.  I added `unsigned` to the `using Base: ` line in `multinverses.jl`.

I can revert this and I could add the `signed(_)` defs in that file, if that is more appropriate.

Reply:
all that is undone